src/test/java/org/tikv/common/ChannelFactoryTest.java

```diff
@@ -59,6 +62,38 @@ public void testCertWatcher() throws InterruptedException {
     assertTrue(changed.get());
   }
 
+  @Test
+  public void testCertWatcherWithExceptionTask() throws InterruptedException {
+    AtomicInteger timesOfReloadTask = new AtomicInteger(0);
+    AtomicInteger timesOfFakeTask = new AtomicInteger(0);
+
+    Executors.newSingleThreadScheduledExecutor()
+        .scheduleAtFixedRate(
+            () -> {
+              timesOfFakeTask.getAndIncrement();
+              System.out.println("Execute fake task for the " + timesOfFakeTask.get() + " times.");
```

Comment:
Could we remove these logs? The logs help little to understand the trace.

Reply:
Rest LGMT.

Reply:
OK. I will remove it.